types/wordpress__editor/wordpress__editor-tests.tsx

```diff
@@ -340,67 +340,73 @@ e.store.name;
 // Store
 // ============================================================================
 
+// Raw dispatch and select don't have type inference at the moment. :(
+// $ExpectType Object
+dispatch(e.store);
+// $ExpectType Object
+select(e.store);
```

Comment:
Previously, we were declaring the types for these manually in the DT package. The built in types unfortunately don't currently have inference.